sale_report_delivered_subtotal/reports/sale_report.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2018 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class SaleReport(models.Model):
+    _inherit = "sale.report"
+
+    price_subtotal_delivered = fields.Float(
+        string='Total Price Delivered',
+    )
+
+    def _query(self, with_clause='', fields={}, groupby='', from_clause=''):
```

Comment:
For avoiding the linter (which is correct that can be dangerous), but with this change it doesn't affect anything:
```suggestion
    def _query(self, with_clause='', fields=None, groupby='', from_clause=''):
```